.coveragerc

```diff
@@ -1,7 +1,7 @@
 
 [run]
 branch = False
-source = Modules,ratlib
+source = Modules,ratlib,database
```

Comment:
Should this be a top-level folder, or a subfolder of `Modules`?

Reply:
It's a package, not part of the Modules package, so it has its own top folder.

`from database import Databasemanager`
vs
`from Modules.database import DatabaseManager`

Reply:
Methinks this should be a sub-package of the `Modules` package. If nothing less its for consistency with the rest of the implementation.

Reply:
You directed me to make this a top level package in the first place?

Reply:
I believe i directed you to make it a package, I do not believe i specified for it to be top level but implied it should match existing filestructure (under `Modules`)

Reply:
I gave this further thought, scratch the above.

The Database module itself isn't a end-functionality module like commands facts are, methinks its present location is fine. the Facts manager should end up under `Modules` as it actually provides functionality that extends to Mecha's public behavior.